pkg/ebpf/errors.go

```diff
@@ -0,0 +1,226 @@
+package ebpf
+
+import (
+	"encoding/binary"
+	"fmt"
+	"math"
+	"strings"
+	"unsafe"
+
+	"github.com/aquasecurity/tracee/pkg/logger"
+)
+
+type BPFErrorType uint32
+
+const (
+	BPF_ERR_MAP_UPDATE_ELEM BPFErrorType = iota + 1
+	BPF_ERR_MAP_DELETE_ELEM
+	BPF_ERR_GET_CURRENT_COMM
+	BPF_ERR_TAIL_CALL
+
+	BPF_ERR_UNSPEC BPFErrorType = math.MaxUint32
+)
+
+func (b BPFErrorType) String() string {
+	stringMap := map[BPFErrorType]string{
```

Comment:
In general these kind of maps should stay out of functions since they are reallocated per call.